lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -133,29 +131,29 @@ def check_assoc(assoc)
           end
         end
 
-        def add_resource_relationships(attrs, serializer, options = {})
-          options[:add_included] = options.fetch(:add_included, true)
-
-          serializer.associations.each do |association|
-            key = association.key
-            serializer = association.serializer
-            opts = association.options
-
-            attrs[:relationships] ||= {}
-
-            if serializer.respond_to?(:each)
-              add_relationships(attrs, key, serializer)
+        def resource_relationship_value(serializer, options = {})
+          if serializer.respond_to?(:each)
+            serializer.map { |s| resource_identifier(s) }
```

Comment:
I'm not sure if we have a test covering this to check if we are following json-api conventions, but would be nice to cover this one:
>...
> Resource linkage MUST be represented as one of the following:
>...
> - an empty array ([]) for empty to-many relationships.